bootstrapper-maven-plugin/src/test/java/io/javaoperatorsdk/BootstrapperTest.java

```diff
@@ -0,0 +1,19 @@
+package io.javaoperatorsdk;
+
+import java.io.File;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class BootstrapperTest {
+
+  Bootstrapper bootstrapper = new Bootstrapper();
+
+  @Test
+  void copiesFilesToTarget() {
+    bootstrapper.create(new File("target"), "io.sample", "test-project");
```

Comment:
Should we check that the generated projects build at least?

Reply:
yes, I plan to add that

Reply:
done.